client/sites/components/sites-dataviews/dataview-style.scss

```diff
@@ -62,4 +62,29 @@
 			}
 		}
 	}
+}
+
+.wpcom-site .main.a4a-layout.sites-dashboard {
+	.a4a-layout__top-wrapper,
+	.a4a-layout__body {
+		> * {
+			// To align with Core's hard-coded device width.
+			// https://github.com/WordPress/gutenberg/blob/ed66cc50e3c0b6785a48c15230c090790c0b0e6c/packages/dataviews/src/components/dataviews/style.scss#L84
+			// TODO: Remove when Core changes to use one of the predefined breakpoints.
+			@media (max-width: 430px) {
+				padding-inline: 24px;
+			}
```

Comment:
@oandregal 
This `430px` is derived from Core. Would it make sense to update it to `$break-mobile: 480px;`? If so, I’m happy to create a PR.

Reply:
I'll defer this one to designers. Can you raise this to them?

Reply:
Thanks! Left a comment for now: https://github.com/WordPress/gutenberg/pull/61333#discussion_r1859672382